src/Plugins/Convert.php

```diff
@@ -0,0 +1,932 @@
+                "usd",
+                "united states dollar",
+                "us dollar",
+                "united states dollars",
+                "us dollars"
+            ],
+            "JPY" => [
+                "jpy",
+                "japanese yen",
+                "jp yen",
+                "yen"
+            ],
+            "BGN" => [
+                "bgn",
+                "bulgarian lev",
+                "bulgarian leva",
+                "lev",
+                "leva"
+            ],
+            "CZK" => [
+                "czk",
+                "czech republican koruna",
+                "koruna",
+                "korun",
+                "korunas",
+                "koruny",
+                "czech koruna",
+                "czech korun",
+                "czech korunas",
+                "czech koruny",
+                "czech republican koruna",
+                "czech republican korun",
+                "czech republican korunas",
+                "czech republican koruny"
+            ],
+            "DKK" => [
+                "dkk",
+                "danish krone",
+                "danish kroner"
+            ],
+            "GBP" => [
+                "gbp",
+                "british pound",
+                "pound",
+                "british pounds",
+                "pounds"
+            ],
+            "HUF" => ["
+                huf",
+                "hungarian forint",
+                "forint",
+                "hungarian forints",
+                "forints"
+            ],
+            "PLN" => [
+                "pln",
+                "polish zloty",
+                "zloty",
+                "zlotys",
+                "zlote",
+                "zlotych",
+                "zloties",
+                "polish zloty",
+                "polish zlotys",
+                "polish zlote",
+                "polish zlotych",
+                "polish zloties"
+            ],
+            "RON" => ["ron",
+                "romanian leu",
+                "romanian lei"
+            ],
+            "SEK" => [
+                "sek",
+                "swedish krona",
+                "swedish kronor"
+            ],
+            "CHF" => [
+                "chf",
+                "swiss franc",
+                "swiss francs"
+            ],
+            "NOK" => [
+                "nok",
+                "norwegian krone",
+                "norwegian kroner"
+            ],
+            "HRK" => [
+                "hrk",
+                "croatian kuna",
+                "kuna",
+                "kunas",
+                "croatian kunas"
+            ],
+            "RUB" => [
+                "rub",
+                "russian ruble",
+                "russian rouble",
+                "russian rubles",
+                "russian roubles"
+            ],
+            "TRY" => [
+                "try",
+                "turkish lira",
+                "turkish liras",
+                "turkish lire"
```

Comment:
I doubt lire is a thing :P

Reply:
Should we also support [`tofurkish`](https://en.wikipedia.org/wiki/Tofurky)?  Some users may be vegetarian.